### ISSUE TITLE
Can set log-level to ERROR

### ISSUE THREAD

**Issue by bsmith1180** [spyderbat/spydertop#5]
According to the `--help` flag, I should be able to set log-level to ERROR,  Doing so generates an exception:

````
$ docker run -it spyderbat/spydertop -i examples/minikube-sock-shop.json.gz --help
Usage: spydertop [OPTIONS] [TIMESTAMP]

...

  --log-level TEXT                What level of verbosity in logs, one of
                                  DEBUG, INFO, WARN, ERROR. Defaults to WARN

$ docker run -it spyderbat/spydertop -i examples/minikube-sock-shop.json.gz --log-level ERROR
Traceback (most recent call last):
  File "/usr/local/bin/spydertop", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/spydertop/cli.py", line 205, in cli
    config = Config(
  File "/usr/local/lib/python3.10/site-packages/spydertop/config.py", line 90, in __init__
    log.log_level = log.LOG_LEVELS.index(log_level)
ValueError: 'ERROR' is not in list

**Comment by spyder-griffith** [comment]
This should have been fixed in 1.0.0, are you sure the docker image is up-to-date? Running with the latest does not show the same behavior for me.

**Comment by spyder-griffith** [comment]
This was confirmed to be an issue with an old version.
